tests/model_fields_/test_embedded_model.py

```diff
@@ -0,0 +1,140 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.test import SimpleTestCase, TestCase
+from django.test.utils import isolate_apps
+
+from django_mongodb_backend.fields import EmbeddedModelField
+
+from .models import (
+    Address,
+    Author,
+    Book,
+    EmbeddedModel,
+    EmbeddedModelFieldModel,
```

Comment:
These are "funny" enough looking for me to suggest maybe add a comment about why `EmbeddedModel` and `EmbeddedModelFieldModel` exist in `models.py` and how they will be used for testing here (aside from the obvious "to test `EmbeddedModelField` and with more detail perhaps).

I'd also prefer practical examples over models with names like `ThisModelExistsJustForThisTest` but probably not a blocker.

Reply:
I agree. That is certainly a wart from copying some code written by others.